test/test_calculate_complexity_scores.py

```diff
@@ -81,8 +97,10 @@ def test_commandline(self):
         # Compare expected results to actual output file
         expected = sbol3.Document()
         expected.read(test_dir / 'test_files' / 'Comparison_file_Complexity_Scores.nt')
+        expected.write('expected.sbol', file_format='turtle')
```

Comment:
This looks like the write was added for debugging purposes.
If you are going to write these, you should test them; if not, delete the debug-only write.